README.md

```diff
@@ -202,6 +202,23 @@ pytest  # run test in current environment
 If you chose to create docs with your plugin read the corresponding docs
 for [Sphinx] or [MkDocs]
 
+### Pre-commit
+
+This template includes a default yaml configuration for [pre-commit](https://pre-commit.com/).
+Among other things, it includes checks for best practices in napari plugins.
```

Comment:
"Among other things" is pretty vague, it might be better to have a short summary of what the pre-commit actually includes (code linting, docstring checks, napari plugin best practice checks). 

If you don't say what it does, people who are not sure might err on the side of caution and not use pre-commit. If you do say what it does, people might think things like auto-linting are useful and worth the extra hassle of figuring out how to use it.

Reply:
i'm torn... this comes back to the "giant wall of text" problem that I feel like we come up against a lot.  Each one of the items in the config _is_ a link to a page describing what it does, so no one is left in the dark if they want to know more.  We can certainly copy a sentence from each of those to summarize here, but the readme is already so long that I feel like people don't read it.  If we tried to summarize the whole of python tooling in our cookiecutter, it would give the impression that making a plugin is really complicated.  It certainly can be, but all this stuff feels like "reference" to me, and not so much front page material

Reply:
I don't want a giant wall of text either. I suspected most of it could be covered by a 3 or 4 item list in a single sentence which is why I suggested it, but if I'm wrong that's ok. (Like, I think multiple checks fall under the two word description "code linting").

If it's not useful, you don't have to do it.